aws/request/connection_reset_error_test.go

```diff
@@ -4,8 +4,10 @@ package request_test
 
 import (
 	"net"
-	"os"
 	"syscall"
 )
 
-var stubConnectionResetError = &net.OpError{Err: &os.SyscallError{Syscall: "read", Err: syscall.ECONNRESET}}
+var (
+	stubConnectionResetErrorAccept = &net.OpError{Op: "accept", Err: syscall.ECONNRESET}
+	stubConnectionResetErrorRead   = &net.OpError{Op: "read", Err: syscall.ECONNRESET}
```

Comment:
One additional change, the appengine, plan9 version of this file should be also be updated with these additional vars as well. Adding the following to `connection_reset_error_other_test.go` should be fine.

```go
var (
     stubConnectionResetErrorAccept = errors.New("accept: connection reset")
     stubConnectionResetErrorRead = errors.New("read: connection reset")
)
```